lib/web_catalog/object_graph_importer.es6.js

```diff
@@ -45,38 +54,116 @@ foam.CLASS({
       required: true,
       final: true,
     },
+    {
+      name: 'numWorkers',
+      value: 16,
+    },
+    {
+      name: 'stubFactory',
+      factory: function() {
+        return this.StubFactorySingleton.create();
+      },
+    },
+    {
+      class: 'FObjectProperty',
+      of: 'org.chromium.apis.web.ApiExtractorService',
+      name: 'apiExtractorService',
+      factory: function() {
+        return this.stubFactory.get(this.ApiExtractorService).create({
+          delegate: this.apiContext_.registry.register(
+            null, null, this.SkeletonBox.create({
+              data: this.ApiExtractorService.create(),
+            })),
+        }, this.containerContext_);
+      },
+    },
+    {
+      class: 'FObjectProperty',
+      of: 'foam.box.Context',
+      name: 'containerContext_',
+      factory: function() {
+        return this.Context.create({
+          unsafe: false,
+          classWhitelist: require('../../data/class_whitelist.json'),
+        });
+      },
+    },
+    {
+      class: 'FObjectProperty',
+      of: 'foam.box.Context',
+      name: 'apiContext_',
+      factory: function() {
+        let ctx = this.Context.create({
+        unsafe: false,
+        classWhitelist: require('../../data/class_whitelist.json'),
+      });
+        ctx.registry = this.BroadcastRegistry.create(null, ctx);
+        const numWorkers = this.numWorkers;
+        let workers = new Array(numWorkers);
+
+        for (let i = 0; i < numWorkers; i++) {
+          // Stub BoxRegistry interface over box addressed to new fork.
+          workers[i] = this.stubFactory.get(this.BoxRegistry).create({
+            delegate: this.ForkBox.create({
+              critical: true,
+              nodeParams:
+              // ['--inspect', '--debug-brk'],
```

Comment:
Don't think you need the linebreak here

Reply:
This was for debugging. Done (next push).